claude/skills/update-ticket/SKILL.md

```
@@ -0,0 +1,102 @@
+- **完了済み**: チケットに書かれているがコードでは既に実装済みの項目
+- **変更あり**: チケットの前提・方針と異なる実装になっている箇所
+- **新規判明**: チケットに記載がないが作業中に判明した事実・追加タスク
+- **未着手**: チケットに書かれていてまだ未実装の項目
+
+### 4. 更新案の提示
+
+乖離箇所をもとに更新案を作成し、ユーザーに提示する。
+
+#### 出力フォーマット
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)